test/sharness/t0081-repo-pinning.sh

```diff
@@ -238,9 +238,9 @@ test_expect_success "some are no longer there" '
 '
 
 test_expect_success "recursive pin fails without objects" '
-	ipfs pin rm "$HASH_DIR1" &&
-	test_must_fail ipfs pin add -r "$HASH_DIR1" --timeout=500ms 2>err_expected8 &&
-	grep "context deadline exceeded" err_expected8 ||
+	ipfs pin rm -r=false "$HASH_DIR1" &&
```

Comment:
it hasnt, its been made more correct. direct pinning used to be the default, so this test removed the direct pin, then added a recursive pin. When the change to make recursive default happened, this bit wasnt fixed. It didnt cause an error because removing a recursive pin will also remove a direct pin if one exists (`rm -r` works on a single file too). So all I did here was make it more explicit that we were removing the direct pin, and not removing a recursive one.